Utils/PluginActivator/source/Module.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#ifndef MODULE_NAME
+#define MODULE_NAME PluginStarter
+#endif
+
+#include <com/com.h>
+#include <core/core.h>
+#include <plugins/Configuration.h>
```

Comment:
As Thunder is deploying  a modular approach, including the module in your code is done by specifying <Module>/<Module>.h This means that if you include the plugins, suggest to use <plugins/plugins.h>